src/gui/integration/integration_test.go

```diff
@@ -1912,9 +1916,222 @@ func TestLiveWalletSpend(t *testing.T) {
 	}
 }
 
+func TestCreateWallet(t *testing.T) {
+	if !doLiveOrStable(t) {
+		return
+	}
+
+	c := gui.NewClient(nodeAddress())
+
+	w, clean := createWallet(t, c)
+	defer clean()
+
+	walletDir := getWalletDirFromEnv(t)
+
+	// Confirms the wallet does exist
+	walletPath := filepath.Join(walletDir, w.GetFilename())
+	_, err := os.Stat(walletPath)
+	require.False(t, os.IsNotExist(err))
+
+	checkWalletEntriesAndLastSeed(t, w)
+}
+
+func TestGetWallet(t *testing.T) {
+	if !doLiveOrStable(t) {
+		return
+	}
+
+	c := gui.NewClient(nodeAddress())
+
+	// Create a wallet
+	w, clean := createWallet(t, c)
+	defer clean()
+
+	// Confirms the wallet can be acquired
+	w1, err := c.Wallet(w.GetFilename())
+	require.NoError(t, err)
+	require.Equal(t, *w, *w1)
+}
+
+func TestGetWallets(t *testing.T) {
+	if !doLiveOrStable(t) {
+		return
+	}
+
+	c := gui.NewClient(nodeAddress())
+
+	// Creates 2 new wallets
+	var ws []wallet.Wallet
+	for i := 0; i < 2; i++ {
+		w, clean := createWallet(t, c)
+		defer clean()
+		// cleaners = append(cleaners, clean)
+		ws = append(ws, *w)
+	}
+
+	// Gets wallet from node
+	readableWallets, err := c.Wallets()
+	require.NoError(t, err)
+
+	// Create the wallet map
+	walletMap := make(map[string]wallet.Wallet)
+	for _, rw := range readableWallets {
+		w, err := rw.ToWallet()
+		require.NoError(t, err)
+		walletMap[w.GetFilename()] = w
+	}
+
+	// Confirms the returned wallets contains the wallet we created.
+	for _, w := range ws {
+		retW, ok := walletMap[w.GetFilename()]
+		require.True(t, ok)
+		require.Equal(t, w, retW)
+	}
+}
+
+// TestWalletNewAddress will generate 30 wallets for testing, and they will
+// be removed automatically after testing.
+//
+// Note: Though the new generated wallet files are all deleted, they are still
+// in the memory, you need to resrat the node to free them.
```

Comment:
I made an issue based off of this, https://github.com/skycoin/skycoin/issues/1086 we can do it later